pkg/azuredisk/fake_azuredisk.go

```diff
@@ -60,6 +60,7 @@ type FakeDriver interface {
 	CSIDriver
 
 	GetSourceDiskSize(ctx context.Context, resourceGroup, diskName string, curDepth, maxDepth int) (*int32, error)
+	GetVolumeLocks() *volumehelper.VolumeLocks
```

Comment:
This should be lower camel cased to keep it package private, i.e. `getVolumeLocks()`, since it is only used by the unit test.

Reply:
Done :)